contracts/deploy/01-outbox/01-arb-to-gnosis-outbox.ts

```diff
@@ -36,17 +36,17 @@ const paramsByChainId = {
     sequencerLimit: 86400, // 24 hours
   },
   HARDHAT: {
-    deposit: parseEther("5"), // 120 xDAI budget for timeout
+    deposit: parseEther("10"), // 120 xDAI budget for timeout
```

Comment:
It looks like the comment doesnt match the code. Am I reading it correctly?

Reply:
fixed